OpenRA.Mods.Common/Traits/Render/WithInfantryBody.cs

```diff
@@ -20,9 +20,15 @@ public class WithInfantryBodyInfo : ITraitInfo, IQuantizeBodyOrientationInfo, IR
 	{
 		public readonly int MinIdleWaitTicks = 30;
 		public readonly int MaxIdleWaitTicks = 110;
-		public readonly string MoveSequence = "run";
-		public readonly string AttackSequence = "shoot";
+
+		[SequenceReference] public readonly string MoveSequence = "run";
+
+		[SequenceReference] public readonly string AttackSequence;
```

Comment:
Can we please refrain from actor- or mod-specific default values in the code?
That has caused enough messes so far.
It is much cleaner to have 10 actor types set their value, even if it is the same, and 2 that use the default empty value, then having to change or disable the default value that might end up being completely unrelated to your mod.
In general I think we should have as few "default" values hardcoded as possible.